src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,646 @@
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CA_CERT="{path-to-cli-ca-cert}"
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CLIENT_CERT="{path-to-cli-client-cert}"
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CLIENT_KEY="{path-to-cli-client-key}"
+~~~
+
+## `molt-lms-cli`
+
+The `molt-lms-cli` command-line interface is used to inspect the LMS instances and [perform cutover](#perform-a-cutover).
+
+To install `molt-lms-cli`, download the binary that matches your system:
+
+- [Intel](https://molt.cockroachdb.com/lms/cli/molt-lms-cli-0.1.0.darwin-arm64.tgz)
+- [ARM](https://molt.cockroachdb.com/lms/cli/molt-lms-cli-0.1.0.darwin-amd64.tgz)
+
+### Commands
+
+|       Command        |                                                                                    Usage                                                                                    |
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `initialize`         | Set up the required objects for running the LMS. You must run this before using the LMS.                                                                                    |
+| `connections list`   | List all client connections to the LMS and their most recent queries.                                                                                                       |
+| `cutover consistent` | Specify a [consistent cutover](#consistent-cutover). You must also specify `begin`, `commit`, or `abort`. For usage details, see [Consistent cutover](#consistent-cutover). |
+| `begin`              | Begin a consistent cutover. This pauses traffic to the source database.                                                                                                     |
+| `commit`             | Commit a consistent cutover. This resumes traffic on the target database. This is only effective after running `cutover consistent begin`.                                  |
+| `abort`              | Abort a consistent cutover after running `consistent cutover begin`, unless you have also run `consistent cutover commit`. This resumes traffic to the source database.     |
+| `status`             | Display the current configuration of the LMS instances.                                                                                                                     |
+
+{% comment %}
+| `cutover immediate`  | Initiate an [immediate cutover](#immediate-cutover). This switches the source of truth to the target database. For usage details, see [Immediate cutover](#immediate-cutover). |
+{% endcomment %}
+
+### Flags
```

Comment:
It might be worth mentioning here that for the orchestrator-url and TLS flags, you can supply the `CLI_ORCHESTRATOR_URL`, `CLI_TLS_CA_CERT`, etc. env vars and export them in the shell in order to not type them every time.

Reply:
Yes, this is already noted in the flag descriptions!